README.md

```diff
@@ -140,6 +140,24 @@ This section applies to Solidity and JavaScript projects alike.
 * Define `files` to minimize the size of the published package.
 * For library projects, `main` should point to `./lib/index.js`.
 
+## Managing internal and external dependencies
```

Comment:
Why did you change the header to be level 2??? Please change it back to be a level 3 header so that it is under the section "package.json".

```suggestion
### Managing internal and external dependencies
```

Reply:
I misunderstood the comment.
Reverted the change. Thank you.